torchtune/models/t5/_model_builders.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torchtune.models.t5._component_builders import t5_encoder
+from torchtune.models.t5._encoder import T5Encoder
+from torchtune.models.t5._tokenizer import T5Tokenizer
+
+
+def t5_v1p1_xxl_encoder(max_seq_len: int = 512) -> T5Encoder:
```

Comment:
thoughts on writing decimal points as `p` instead of `_` in snake case? imo it's hard to read the `_` decimals:

`t5_v1_1_xxl_encoder` vs `t5_v1p1_xxl_encoder`

Reply:
I think the p could be better. But I'd want to switch everything to this notation then

Reply:
I'll make a separate PR for this

Reply:
Can you use the _ notation for now and just update this along with everything else in that PR?